x-pack/plugins/reporting/server/lib/enqueue_job.ts

```diff
@@ -23,50 +22,45 @@ export function enqueueJobFactory(
   reporting: ReportingCore,
   parentLogger: LevelLogger
 ): EnqueueJobFn {
-  const logger = parentLogger.clone(['queue-job']);
-  const config = reporting.getConfig();
-  const jobSettings = {
-    timeout: durationToNumber(config.get('queue', 'timeout')),
-    browser_type: config.get('capture', 'browser', 'type'),
-    max_attempts: config.get('capture', 'maxAttempts'),
```

Comment:
Priority was unused, and can be untouched with no effect.

Timeout, Browser Type, and Max Attempts should be pulled by the server when it starts processing the report job, not when it creates the pending report.